### DISCUSSION TITLE
How do I uninstall deno from my system?

### DISCUSSION THREAD

**Discussion by TelegramUsername** [denoland/deno#18488] [Q&A]
I wanted to uninstall Deno from my system. how do I do it?

**Comment by markc** [comment]
Simply remove the "deno" binary and the project folder where you tested some deno scripts. There will also be some cached files, but where they are depends on your OS. On linux, my deno binary is /usr/bin/deno, my project devel folder is ~/Dev and there are deno related files in ~/.deno and ~/.cache/deno.

**Reply by mak448a** [reply]
Make sure to also remove deno from your ~/.bashrc
